src/libs/SDLWindowManager/SDLWindowManager.cpp

```diff
@@ -0,0 +1,420 @@
+#include "SDLWindowManager.hpp"
+
+#include <SDL.h>
+#include <SDL_system.h>
+#include <SDL_syswm.h>
+#include <SDL_video.h>
+#include <map>
+
+namespace storm
+{
+namespace
+{
+// TODO: Use scancodes everywhere instead of Windows virtual key
+static const std::map<unsigned int, unsigned int> VK_TO_SDL_MAP{
```

Comment:
It seems, window management and input management got mixed into one class. I guess it's better to keep them separated